keyboards/era/linx3/fave65s/keyboard.json

```
@@ -0,0 +1,766 @@
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "jellybean_raindrops": true,
+            "pixel_flow": true,
+            "pixel_fractal": true,
+            "pixel_rain": true,
+            "rainbow_beacon": true,
+            "rainbow_moving_chevron": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "riverflow": true,
+            "starlight": true,
+            "starlight_dual_hue": true,
+            "starlight_dual_sat": true
+        },
+        "default": {
+            "animation": "rainbow_moving_chevron"
+        },
+        "driver": "ws2812",
+        "layout": [
+            {"matrix": [2, 0], "x": 10, "y": 39, "flags": 8},
+            {"x": 0, "y": 0, "flags": 2},
+            {"x": 13, "y": 0, "flags": 2},
+            {"x": 26, "y": 0, "flags": 2},
+            {"x": 40, "y": 0, "flags": 2},
+            {"x": 53, "y": 0, "flags": 2},
+            {"x": 66, "y": 0, "flags": 2},
+            {"x": 79, "y": 0, "flags": 2},
+            {"x": 92, "y": 0, "flags": 2},
+            {"x": 105, "y": 0, "flags": 2},
+            {"x": 119, "y": 0, "flags": 2},
+            {"x": 132, "y": 0, "flags": 2},
+            {"x": 145, "y": 0, "flags": 2},
+            {"x": 158, "y": 0, "flags": 2},
+            {"x": 171, "y": 0, "flags": 2},
+            {"x": 184, "y": 0, "flags": 2},
+            {"x": 198, "y": 0, "flags": 2},
+            {"x": 211, "y": 0, "flags": 2},
+            {"x": 224, "y": 0, "flags": 2},
+            {"x": 224, "y": 13, "flags": 2},
+            {"x": 224, "y": 26, "flags": 2},
+            {"x": 224, "y": 38, "flags": 2},
+            {"x": 224, "y": 51, "flags": 2},
+            {"x": 224, "y": 64, "flags": 2},
+            {"x": 211, "y": 64, "flags": 2},
+            {"x": 198, "y": 64, "flags": 2},
+            {"x": 184, "y": 64, "flags": 2},
+            {"x": 171, "y": 64, "flags": 2},
+            {"x": 158, "y": 64, "flags": 2},
+            {"x": 145, "y": 64, "flags": 2},
+            {"x": 132, "y": 64, "flags": 2},
+            {"x": 119, "y": 64, "flags": 2},
+            {"x": 105, "y": 64, "flags": 2},
+            {"x": 92, "y": 64, "flags": 2},
+            {"x": 79, "y": 64, "flags": 2},
+            {"x": 66, "y": 64, "flags": 2},
+            {"x": 53, "y": 64, "flags": 2},
+            {"x": 40, "y": 64, "flags": 2},
+            {"x": 26, "y": 64, "flags": 2},
+            {"x": 13, "y": 64, "flags": 2},
+            {"x": 0, "y": 64, "flags": 2},
+            {"x": 0, "y": 51, "flags": 2},
+            {"x": 0, "y": 38, "flags": 2},
+            {"x": 0, "y": 26, "flags": 2},
+            {"x": 0, "y": 13, "flags": 2}
+        ],
+        "sleep": true
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0011",
+        "vid": "0x4552"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP19"
+    },
+    "layouts": {
```

Add community layout support
```suggestion
    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_split_bs", "65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs", "65_iso_blocker", "65_iso_blocker_split_bs", "65_iso_blocker_tsangan", "65_iso_blocker_tsangan_split_bs"],
    "layouts": {
```

---

I applied it. Thanks for your help.